frontend/package.json

```diff
@@ -37,7 +37,7 @@
     "build": "if [ \"${KIALI_ENV}\" = \"production\" ]; then npm run build:prod; else npm run build:dev; fi",
     "build-css": "node-sass src/ --output-style compressed --include-path $npm_package_sassIncludes_src -o src/",
     "build:dev": "sh -ac '. ./.env.upstream; npm run lint && npm run build:kiali'",
-    "build:kiali": "npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build",
+    "build:kiali": "npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) GENERATE_SOURCEMAP=false EXTEND_ESLINT=true react-scripts build --profile",
```

Comment:
Is there any way to add this to `build:dev` without adding to `build:prod`? Not if the CI build is done with `build:dev` or `build:prod`.

Reply:
the CI is a prod build.  I think if it were a dev build this may not have been an issue, but I'm not sure we want a dev build.  If we are not comfortable adding this option to a prod build, then I'd probably suggest we add a CI build option.  From what I read, there is not a lot of additional overhead.

Reply:
A separate CI build opt sounds fine. It can either be included in this PR or in a followup since the graph tests are causing the rest of the CI runs to fail at the moment.

Reply:
let's do it in a follow up (and maybe someone else that isn't as much of a dork at that stuff as me) :-)  Would like to get this in.

Reply:
so, if you are ok, feel free to approve.

Reply:
oh, it's already approved  :-)  I'll merge, thanks.  And I'll create the follow-up issue.